chart/values.yaml

```diff
@@ -715,6 +715,113 @@ webserver:
             topologyKey: "kubernetes.io/hostname"
   tolerations: []
 
+# Airflow Triggerer Config
+triggerers:
+  # Number of airflow triggerers in StatefulSet
+  replicas: 1
+
+  # Command to use when running Airflow triggerers (templated).
+  command: ~
+  # Args to use when running Airflow triggerers (templated).
+  args:
+    - "bash"
+    - "-c"
+    # The format below is necessary to get `helm lint` happy
+    - |-
+      exec \
+      airflow triggerer
+
+  # Update Strategy when triggerers are deployed as a StatefulSet
+  updateStrategy: ~
+  # Update Strategy when triggerers are deployed as a Deployment
+  strategy:
+    rollingUpdate:
+      maxSurge: "100%"
+      maxUnavailable: "50%"
+
+  # Create ServiceAccount
+  serviceAccount:
+    # Specifies whether a ServiceAccount should be created
+    create: true
+    # The name of the ServiceAccount to use.
+    # If not set and create is true, a name is generated using the release name
+    name: ~
+
+    # Annotations to add to triggerer kubernetes service account.
+    annotations: {}
+
+  persistence:
+    # Enable persistent volumes
+    enabled: false
```

Comment:
Should this default to True like workers?

Reply:
I made it a deployment -- there isn't any state on Triggers